test/uk/gov/hmrc/mtdsatestsupportapi/models/request/createTestBusiness/BusinessSpec.scala

```diff
@@ -169,7 +183,20 @@ class BusinessSpec extends UnitSpec with CreateTestBusinessFixtures {
   private def businessWithMinimumFields(typeOfBusiness: TypeOfBusiness): Business = {
     Business(
```

Comment:
```suggestion
    // @formatter:off
    Business(
      typeOfBusiness = typeOfBusiness,
      None, None, None, None, None, None, None,
      None, None, None, None, None, None, None
    )
    // @formatter:on
```

think this makes it easier to see that every field apart from typeOfBusiness is an optional set to None